SYCL/Basic/queue/queue.cpp

```diff
@@ -23,7 +23,7 @@ void print_queue_info(const queue &q) {
                  q.get_context().get_platform().get_backend() !=
                      cl::sycl::backend::opencl)
                     ? nullptr
-                    : q.get())
```

Comment:
The test should include `<CL/sycl/backend/opencl.hpp>` now.
This is relevant for other modified tests also.
I'm not sure about this, but, probably, the test should be linked against OpenCL library also.